dre-service/src/main/java/io/cdap/re/RulesDB.java

```diff
@@ -22,6 +22,7 @@
 import com.google.gson.JsonPrimitive;
 import com.google.gson.reflect.TypeToken;
 import io.cdap.cdap.api.dataset.lib.CloseableIterator;
+import io.cdap.cdap.proto.id.NamespaceId;
```

Comment:
don't re-use this class. This represents a namespace in the cdap rest api and should not be re-purposed here.

Reply:
Done